lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -90,6 +90,8 @@ export class ResponseHandler {
             if (!StringUtils.isEmpty(this.clientInfo.uid) && !StringUtils.isEmpty(this.clientInfo.utid)) {
                 this.homeAccountIdentifier = `${this.clientInfo.uid}.${this.clientInfo.utid}`;
             }
+        } else {
+            this.homeAccountIdentifier = "";
```

Comment:
Should we attempt to use `idToken.claims.oid || idToken.claims.sub` as a fallback in this scenario? And then if that doesn't exist, make it blank?

Reply:
Somewhat related to my comment below about falling back to cached id token.

Reply:
Chatted offline, will split validateTokenResponse into two functions.

Reply:
After further looking into this and hearing back about scenarios where client_info can be expected, it is not necessary that to make this change. Currently, `ResponseHandler` checks that client_info was returned when: 
- Id token was returned and is not an ADFS authority. 

In these cases, client_info_empty_error will be thrown if no client_info was returned.